tasks/system_probe.py

```diff
@@ -298,6 +298,7 @@ def build_object_files(ctx, bundle_ebpf=False):
         '-D__KERNEL__',
         '-DCONFIG_64BIT',
         '-D__BPF_TRACING__',
+        '-DKBUILD_MODNAME="\\"foo\\""',
```

Comment:
What is this for ?

Reply:
While building the system probe, the build was complaining of this missing var during the build of the sysprobe module since recent changes, this fixes it https://github.com/DataDog/datadog-agent/pull/6254/commits/528f52fd281c32852915d97d660b25b69d076577